src/renderer/canvas/edge/shared.ts

```diff
@@ -10,3 +10,7 @@ export interface IEdgeArrow {
   angle: number;
   length: number;
 }
+
+export const DEFAULT_SOLID_LINE_PATTERN: number[] = [];
+export const DEFAULT_DASHED_LINE_PATTERN: number[] = [5, 5];
+export const DEFAULT_DOTTED_LINE_PATTERN: number[] = [1, 1];
```

Comment:
Maybe you will need to move this closer to the `models/edge` - check the other comment on where the code around patterns should be.